src/nimscripter/nimscripter.nim

```diff
@@ -208,3 +206,11 @@ proc invoke*(intr: Interpreter, procName: string, argBuffer: string = "", T: typ
   when T isnot void:
     var pos: BiggestInt
     getFromBuffer(ret.strVal, T, pos)
+
+proc invoke*(intr: Interpreter, procName: string, argBuffer: string = "", T: typeDesc = void): T =
+  let
+    foreignProc = intr.selectRoutine(procName & "Exported")
+  result = invoke(intr, foreignProc, argBuffer, T)
+
+proc selectRoutine*(intr: Interpreter, procName: string): PSym =
```

Comment:
Move this above the `string` invoke and add `{.inline.}` to the string invoke and select routine.

Reply:
done